### ISSUE TITLE
About FiLM

### ISSUE THREAD

**Issue by TengNN** [AVC2-UESTC/ExVC#1]
Hello, after reading your article, I also tried to use FiLM, but the loss does not converge? What is the reason for this?
I am doing it based on FreeVC framework, should I train for longer?

**Comment by TKsavy** [comment]
Hi @TengNN, 

Thanks for your feedback! In our experiments, we used FiLM with ExVC. I'm not entirely sure if it would produce similar results with FreeVC, as their architectures differ slightly. We'll need to run additional experiments to confirm this.   

By the way, how long did you train your FreeVC model? Did you follow the same recipe (dataset and other hyperparameters) as the original FreeVC model? If possible, could you share your training loss curve with us?
